.github/workflows/run_tests.yml

```diff
@@ -116,3 +115,39 @@ jobs:
         run: | 
           mkdir -p doc/source/_static
           sphinx-build -EnW -b html doc/source/ doc/build/html
+
+  deploy:
```

Comment:
Isn't this double now i.e. defined in deploy.yml and here?

Reply:
No, this one deploys to test.pypi.org rather than actual pypi.org. I would like to deduplicate these 2 files somewhat, but we only want to properly deploy on release, rather than for every PR.

Reply:
ok!